extra/resources/ClusterMon

```diff
@@ -153,7 +153,8 @@ ClusterMon_monitor() {
     if [ -f $OCF_RESKEY_pidfile ]; then
 	pid=`cat $OCF_RESKEY_pidfile`
 	if [ ! -z $pid ]; then
-	    kill -s 0 $pid >/dev/null 2>&1; rc=$?
+	    ps -wwfp $pid | grep $OCF_RESKEY_pidfile >/dev/null 2>&1 && \
+	        kill -s 0 $pid >/dev/null 2>&1; rc=$?
```

Comment:
Sorry, I may be missing couple of things:
- agents arguments are present as env. variables rather than command arguments, correct?
- how can you be sure that pidfile specification didn't change in between?